.github/wordlist.txt

```diff
@@ -57,12 +57,16 @@ geopoint
 geospatial
 glpk
 gpd
+gurobi
+hardcoded
 inplace
+integrality
 ipynb
 ipywidgets
 iterable
 json
 libs
+MIP
```

Comment:
Minor point, but words starting with first letter capitalized are all up top in a separate section, can you please put MIP alphabetically in that section?

Reply:
Good catch